internal/privatemessaging/recipients.go

```diff
@@ -105,6 +106,17 @@ func (pm *privateMessaging) resolveNode(ctx context.Context, org *fftypes.Organi
 }
 
 func (pm *privateMessaging) getReceipients(ctx context.Context, in *fftypes.MessageInOut) (gi *fftypes.GroupIdentity, err error) {
```

Comment:
Not your change, but there's a typo in this method name.

Reply:
fixed